library/templates/doitt_buildingfootprints.yml

```diff
@@ -4,7 +4,7 @@ dataset:
   acl: public-read
   source:
     socrata:
-      uid: w3t7-k3ej
+      uid: uic8-njst
       format: csv   
```

Comment:
why is this one using csv but the historical one is using geojson?

Reply:
@SPTKL That's how we previously pulled the two datasets using the recipe app.